docs/source/publish_and_share_kedro_viz_on_azure.md

```diff
@@ -0,0 +1,121 @@
+
+![](./images/azure_member_assign.png)
+
+## Step 3: Set credentials
+
+Once that's completed, you'll need to set your Azure credentials as environment variables in your terminal window, as shown below:
+
+```bash
+export AZURE_STORAGE_TENANT_ID="your-app-tenant-id"
+export AZURE_STORAGE_CLIENT_ID="your-app-client-id"
+export AZURE_STORAGE_CLIENT_SECRET="your-app-client-secret-value"
+```
+
+For more information, see the official Azure documentation about [how to work with environmental credentials](https://learn.microsoft.com/en-us/dotnet/api/azure.identity.environmentcredential?view=azure-dotnet).
+
+```{note}
+Kedro-Viz uses Gen2 filesystem protocol `abfs` to write files on AzureBlobStorage.
+```
+
+```{important}
+Having a `$web` container in your AzureBlobStorage is mandatory to use Kedro-Viz publish and share feature on Azure. For more information, see the official Azure documentation about 
+[Setting up a static website](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blob-static-website#setting-up-a-static-website).
+```
+
+### Permissions and access control
+
+All permissions and access control are controlled by Azure. It's up to you, the user, if you want to allow anyone to see your project or limit access to certain IP addresses, users, or groups.
+
+You can control who can view your visualisation using [attribute-based access control](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-auth-abac). See the official Azure documentation for more information.
+
+### Billing
+
+You pay for storing objects on your AzureBlobStorage. The amount you pay depends on the volume of data stored per month, quantity and types of operations performed, along with any data transfer costs, data redundancy option selected.
+
+See the official [Azure documentation](https://azure.microsoft.com/en-us/pricing/details/storage/blobs/) for more information.
+
+
+## Step 4: Publish and share the project
+Once your Cloud storage is configured and the credentials are set, you are now ready to publish and share your Kedro-Viz project. Start Kedro-Viz by running the following command in your terminal:
+
+```bash
+kedro viz run
+```
+
+Click the **Publish and share** icon in the lower-left of the application. You will see a modal dialog to select your relevant AWS Bucket Region and enter your Bucket Name.
+
+```{note}
+From Kedro-Viz version 8.0.0, you will see a modal dialog to select your hosting platform, input your bucket name and endpoint link. The endpoint link can be found under **Storage account -> Capabilities -> Static website -> Primary endpoint**.
+```
+
+Once those details are complete, click **Publish**. A hosted, shareable URL will be returned to you after the process completes.
+
+Here is an example of the flow (TODO - Need to add flows specific to cloud provider):
+
+![](./images/kedro-publish-share.gif)
+
+```{note}
+We will be updating the user flow doc for v8.0.0 soon...
+```
```

Comment:
we can just have the v8.0.0 flow for azure and gcp as there was no support before. But for AWS, we can have both the flows

Reply:
Resolved.